packages/shared/src/components/post/smartPrompts/PromptButtons.tsx

```diff
@@ -78,6 +81,19 @@ export const PromptButtons = ({
   const promptsCount = prompts?.length || 0;
   const remainingTags = promptsCount - promptList?.length;
 
+  const onPromptClick = useCallback(
+    (id) => {
+      if (isMobile) {
+        openModal({
+          type: LazyModal.SmartPrompt,
+        });
+        return;
+      }
+      setActivePrompt(id);
+    },
+    [isMobile, openModal, setActivePrompt],
+  );
```

Comment:
Only show modal if not plus 🙈 
![image](https://github.com/user-attachments/assets/ae1d9fd1-6e98-40ec-b4fe-b963c43ccb7e)

Reply:
Oh yeah good point XD